### ISSUE TITLE
Coverage data

### ISSUE THREAD

**Issue by ejgallego** [ejgallego/coq-universe#35]
We may want to use coq-universe CI to run coverage data.

cc @skyskimmer

**Comment by Alizter** [comment]
Hopefully at some point in dune we can enable instrumentation with an `env` stanza which will let us do this easily.

**Comment by ejgallego** [comment]
@Alizter yes! I'd be great if we could reuse your very nice work on the Dune repos.

I hope to fix the current problem with -boot soon
